recipe/meta.yaml

```diff
@@ -12,14 +12,17 @@ source:
   url:
     - https://cran.r-project.org/src/contrib/RNetCDF_{{ version }}.tar.gz
     - https://cran.r-project.org/src/contrib/Archive/RNetCDF/RNetCDF_{{ version }}.tar.gz
-  sha256: 7d5a1e47ba3f41050e92ac27fe12c9ffd8dea67e132c43ff071dcdb0fad4ae45
+  sha256: d513abfefe725f4253287ca61e507e1770c6a2cfab2cd2cd00d58de521a5adc5
 
 build:
   merge_build_host: True  # [win]
   number: 1004
   script:
     - R CMD INSTALL --build .  # [win]
-    - R CMD INSTALL --configure-args="CPPFLAGS=-I$PREFIX/include LDFLAGS=-L$PREFIX/lib" --build .  # [not win]
+    - export LD_LIBRARY_PATH=${PREFIX}/lib:$LD_LIBRARY_PATH  # [unix]
+    - export DYLD_LIBRARY_PATH=${PREFIX}/lib:$DYLD_LIBRARY_PATH  # [osx]
+    - R CMD INSTALL --configure-args="CPPFLAGS=-I${PREFIX}/include LDFLAGS=-L${PREFIX}/lib" --build .  # [unix]
```

Comment:
You shouldn't override `CPPFLAGS` and `LDFLAGS` like this. Use the env variables

Reply:
To be very honest, I don't think those ``CPPFLAGS=-I$PREFIX/include LDFLAGS=-L$PREFIX/lib`` are necessary since conda already set those. I think we may try to remove it and it should be fine.

I didn't remove it yet because I am trying to compile it on all plataforms first

Reply:
You are running into the errors and need `LD_LIBRARY_PATH` because LDFLAGS doesn't have rpath in it, but the value set by the compilers has them.

Reply:
I see, thanks for the explanation
Let's see if it passes now and I will remove the ``LD_LIBRARY_PATH`` as well :)